core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -213,8 +217,12 @@ private void parseArguments(final String[] args) {
             .ofType(String.class)
             .describedAs("id")
             .required();
-        bootstrapServerOption = optionParser.accepts("bootstrap-servers", "Comma-separated list of broker urls with format: HOST1:PORT1,HOST2:PORT2")
-            .withRequiredArg()
+        bootstrapServerOption = optionParser.accepts("bootstrap-server", "The server(s) to use for bootstrapping.")
```

Comment:
Should we use a description that's similar to the one in the console producer, which also had a KIP to deprecate a different property in favor of `--bootstrap-server`?

Thinking we can copy and tweak from here: https://github.com/apache/kafka/blob/7b49f175b964c468d5ca7ac9bd70dfd72e6a2e4e/core/src/main/scala/kafka/tools/ConsoleProducer.scala#L135

Reply:
Thanks for a hint. Apply code you suggested.